yarn-project/p2p/package.json

```diff
@@ -51,23 +51,33 @@
     "@aztec/circuits.js": "workspace:^",
     "@aztec/foundation": "workspace:^",
     "@aztec/kv-store": "workspace:^",
-    "@chainsafe/discv5": "^9.0.0",
-    "@chainsafe/enr": "^3.0.0",
+    "@chainsafe/discv5": "9.0.0",
+    "@chainsafe/enr": "3.0.0",
+    "@chainsafe/libp2p-gossipsub": "13.0.0",
     "@chainsafe/libp2p-noise": "^15.0.0",
     "@chainsafe/libp2p-yamux": "^6.0.2",
-    "@libp2p/bootstrap": "^9.0.4",
-    "@libp2p/crypto": "^4.0.3",
-    "@libp2p/identify": "^1.0.15",
-    "@libp2p/interface": "^1.1.4",
-    "@libp2p/interface-libp2p": "^3.2.0",
-    "@libp2p/kad-dht": "^10.0.4",
-    "@libp2p/mplex": "^10.0.16",
-    "@libp2p/peer-id": "^4.0.7",
-    "@libp2p/peer-id-factory": "^4.0.7",
-    "@libp2p/tcp": "^9.0.16",
-    "@multiformats/multiaddr": "^12.1.14",
+    "@libp2p/bootstrap": "10.0.0",
+    "@libp2p/crypto": "4.0.3",
+    "@libp2p/identify": "1.0.18",
+    "@libp2p/interface": "1.3.1",
+    "@libp2p/kad-dht": "10.0.4",
+    "@libp2p/mplex": "10.0.16",
+    "@libp2p/peer-id": "4.0.7",
+    "@libp2p/peer-id-factory": "4.1.1",
+    "@libp2p/peer-store": "10.0.16",
+    "@libp2p/tcp": "9.0.24",
+    "@multiformats/multiaddr": "12.1.14",
+    "interface-datastore": "^8.2.11",
+    "interface-store": "^5.1.8",
+    "it-all": "^3.0.4",
```

Comment:
As mentioned elsewhere, can we remove these dependencies? (The ones not directly required by libp2p etc)